apps/google-analytics-4/frontend/src/components/main-app/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import tokens from '@contentful/f36-tokens';
+
+const ACCESSIBILITY_LABEL = 'Analytics line chart';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip);
+
+ChartJS.defaults.font.size = 12;
+ChartJS.defaults.font.family = tokens.fontStackPrimary;
+ChartJS.defaults.font.weight = '500';
+ChartJS.defaults.borderColor = tokens.gray200;
+ChartJS.defaults.datasets.line.borderColor = tokens.colorPrimary;
+
+interface LineChartProps {
+  dataValues: Array<string | number>;
+  xAxisLabels: Array<string>;
+  tooltipMetricLabel: string;
+  accessibilityLabel: string;
+}
+
+const LineChart = ({
+  dataValues,
+  xAxisLabels,
+  tooltipMetricLabel,
+  accessibilityLabel,
+}: LineChartProps) => {
+  const data = {
+    labels: xAxisLabels,
+    datasets: [
+      {
+        data: dataValues,
+      },
+    ],
+  };
+
+  const options = {
```

Comment:
Nit: Looks like chartJS provides a `CoreChartData` type that we might be able to use here: https://github.com/chartjs/Chart.js/blob/cfefb3d/src/types/index.d.ts#L1461 Might help for type safety so we aren't passing invalid values to any of these properties.

Reply:
GREAT catch! Thank you! Will absolutely add this 💯